meta_capstone/ViewControllers/InGameViewController.m

```diff
@@ -106,6 +111,36 @@ -(void)checkUpdate {
     }
 }
 
+-(void)checkHost {
+    self.game = [[PFQuery queryWithClassName:@"Game"] getObjectWithId:self.game.objectId];
+    if(self.game[@"requestingHost"] != nil) {
+        PFObject *requestingUser = [[PFQuery queryWithClassName:@"AppUser"] getObjectWithId:self.game[@"requestingHost"]];
+        NSString *info = [NSString stringWithFormat:@"%@ is requesting host. Would you like to give them control of the board?", requestingUser[@"name"]];
+        UIAlertController* alert = [UIAlertController alertControllerWithTitle:@"Host Request"
+                                       message:info
+                                       preferredStyle:UIAlertControllerStyleAlert];
+        UIAlertAction* accept = [UIAlertAction actionWithTitle:@"Yes" style:UIAlertActionStyleDefault
+           handler:^(UIAlertAction * action) {
+            NSLog(@"GIVING HOST");
+            [self.game removeObjectForKey:@"requestingHost"];
+            [self.game save];
+            [alert dismissViewControllerAnimated:YES completion:nil];
+        }];
+        
+        UIAlertAction* deny = [UIAlertAction actionWithTitle:@"No" style:UIAlertActionStyleDefault
+           handler:^(UIAlertAction * action) {
+            NSLog(@"DENIED HOST");
+            [self.game removeObjectForKey:@"requestingHost"];
+            [self.game save];
+            [alert dismissViewControllerAnimated:YES completion:nil];
+        }];
+        
+        [alert addAction:accept];
+        [alert addAction:deny];
+        [self presentViewController:alert animated:YES completion:nil];
```

Comment:
This UIAlert is ok for now, but it were being re-used in anyway it would need to be in a helper method. Also, there is a way to condense the  instantiation of the controller so the actions and alert are created together and not separately for a cleaner look, but not a blocker.